lib/tasks/projects.rake

```diff
@@ -28,24 +28,26 @@ namespace :projects do
   task :check_status, %i[max_num_of_projects_to_check batch_size] => :environment do |_task, args|
     exit if ENV["READ_ONLY"].present?
 
-    # default should spread out around 50k package jobs per platform over ~ 83 minutes
-    max_num_of_projects_to_check = args.max_num_of_projects_to_check.nil? ? 50000 : args.max_num_of_projects_to_check.to_i
-    batch_size = args.batch_size.nil? ? 5000 : args.batch_size.to_i
-
-    PLATFORMS_FOR_STATUS_CHECKS.each do |platform|
-      project_ids_to_check = Project.platform(platform).not_removed.where("projects.updated_at < ?", 1.week.ago).order(updated_at: :desc).select("id")
-      enqueued_projects_sum = 0
-      project_ids_to_check.limit(max_num_of_projects_to_check).in_batches(of: batch_size) do |project_ids_batch|
-        project_ids_batch.pluck(:id).each_with_index do |project_id, i|
-          CheckStatusWorker.perform_in(i, project_id)
-        end
+    max_num_of_projects_to_check = args.max_num_of_projects_to_check.nil? ? 150000 : args.max_num_of_projects_to_check.to_i
+    batch_size = args.batch_size.nil? ? 10000 : args.batch_size.to_i
+
+    project_ids_to_check = Project
+      .where(platform: PLATFORMS_FOR_STATUS_CHECKS)
```

Comment:
Thoughts about keeping the filter for looking at things older than 1 week per existing clause on line 36?

Reply:
I can do that, though given the volume of items to check, I don't *think* that really changes anything in terms of what's processed. But it won't really hurt either way, so I'll put it back!